src/libsync/theme.cpp

```diff
@@ -132,6 +132,29 @@ QUrl Theme::stateOfflineImageSource() const
     return imagePathToUrl(themeImagePath("state-offline", 16));
 }
 
+QString Theme::trayHeaderThemeImagePath(const QString &iconName) const {
+    const bool headerHasDarkBg = isDarkColor(wizardHeaderBackgroundColor());
+    const QString imagePath = QString::fromLatin1(":/client/theme/%1/%2.%3").arg(headerHasDarkBg? "white" : "black").arg(iconName).arg("svg");
```

Comment:
@camilasan 
If it's always "svg", maybe remove "%3" and use ".svg" in it's place, and then also get rid of .arg("svg");?